ray-curator/ray_curator/stages/video/io/video_reader.py

```diff
@@ -47,7 +48,8 @@ class VideoReaderStage(ProcessingStage[FileGroupTask, VideoTask]):
     Note:
         Currently supports local filesystem paths only. S3 support is planned for future releases.
```

Comment:
We support it now :-) 

```suggestion
        Currently supports local filesystem paths only
```